### PR TITLE
Feat/pulse package

### PR DESCRIPTION
**Pull request by IvaDey**
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Feature | No          |
| Doc | No          |

---

The agenda package seems not to be maintained anymore (the last commit was over 2 years ago).

There is a package for integration with [@pulsecron/pulse](https://github.com/pulsecron/pulse) – a fresh fork of the agenda.

## Todos

- [x] Tests
- [x] Documentation

### CODE REVIEW

File: docs/tutorials/pulse.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    await this.pulse.schedule("in 2 hours", "email.sendFollowUpEmail", {user});
+
+    return user;
+  }
+}
+```
+
+## Authors
+
+<GithubContributors :users="['ochrstn', 'xCryzed']"/>
```
Comment (Romakita) [inline docs/tutorials/pulse.md]: You can add your github login ;) while pulse package is your initiative!

**Review by Romakita — APPROVED** [review]

File: docs/tutorials/pulse.md (hunk 1 above)
Comment (IvaDey) [reply docs/tutorials/pulse.md]: Added, thanks!)

Comment (IvaDey) [reply docs/tutorials/pulse.md]: Is it okay with an additional non-conventional commit?

Comment (Romakita) [reply docs/tutorials/pulse.md]: You can use docs or chore in this case. But this change impact the release note ;). So it's fine

**Pull request merged**